src/card__rating.css

```diff
@@ -0,0 +1,42 @@
+.card__rating {
+  width: 166px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 0 auto 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars .stars__star:nth-child(5n) {
+  margin-right: 0;
+}
+
+.card__rating-reviews {
+  height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-accent);
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
```

Comment:
```suggestion
```

Reply:
Same. move to the card.css

Reply:
And rename this file card__rating.css => stars.css